porcupine/menubar.py

```diff
@@ -280,18 +296,23 @@ def setup():
     easier.
     """
 
-    def update_enabledness(*junk: object) -> None:
+    def update_enabledness(*junk: object, path: str) -> None:
         tab = get_tab_manager().select()
+
         parent, child = _split_parent(path)
         menu = get_menu(parent)
         index = _find_item(menu, child)
         if index is None:
             raise LookupError(f"menu item {path!r} not found")
+        if not tab:
+            menu.entryconfig(index, state="disabled")
+            return
+
         menu.entryconfig(index, state=("normal" if callback(tab) else "disabled"))
```

Comment:
```suggestion
        if tab and callback(tab):
            menu.entryconfig(index, state="normal")
        else:
            menu.entryconfig(index, state="disabled")
```

A bit cleaner

Reply:
probably will do `tab is not None and callback(tab)`, as `None` is what we are actually checking for and that is the more pythonic (I believe) way to check for that.

Reply:
Yes, we are actually checking for `not None`, but I think it is not necessary to explicitly check for that, because a `FileTab` widget will never be "falsy" anyway.

Reply:
`tab` can be none, I found this the hard way.

Reply:
Suggesting the explicit thing that we all know will work:

```suggestion
        if tab is not None and callback(tab):
            menu.entryconfig(index, state="normal")
        else:
            menu.entryconfig(index, state="disabled")
```